es/common.go

```diff
@@ -29,7 +31,7 @@ var (
 
 func InitVars() {
 	pythonPath = viper.GetString("elasticsearch.python-path")
-	operatingSystem = viper.GetString("elasticsearch.os")
+	operatingSystem = viper.GetString("mdb.os")
```

Comment:
We don't really need a config param of our own.
We can let the [runtime pkg](https://socketloop.com/tutorials/golang-detect-os-operating-system) do that for us.

Reply:
Done.